tests/sles4sap/qesapdeployment/configure.pm

```diff
@@ -89,15 +89,17 @@ sub run {
         $variables{HANA_LOG_DISK_TYPE} = get_var('QESAPDEPLOY_HANA_DISK_TYPE', 'pd-ssd');
     }
 
-    if (check_var('PUBLIC_CLOUD_PROVIDER', 'AZURE')) {
-        my %peering_settings = qesap_az_calculate_address_range(slot => get_required_var('WORKER_ID'));
-        $variables{VNET_ADDRESS_RANGE} = $peering_settings{vnet_address_range};
-        $variables{SUBNET_ADDRESS_RANGE} = $peering_settings{subnet_address_range};
-        if ($variables{FENCING} eq 'native') {
-            $variables{AZURE_NATIVE_FENCING_AIM} = get_var('QESAPDEPLOY_AZURE_FENCE_AGENT_CONFIGURATION', 'msi');
-            if ($variables{AZURE_NATIVE_FENCING_AIM} eq 'spn') {
-                $variables{AZURE_NATIVE_FENCING_APP_ID} = get_var('QESAPDEPLOY_AZURE_SPN_APPLICATION_ID', get_required_var('_SECRET_AZURE_SPN_APPLICATION_ID'));
-                $variables{AZURE_NATIVE_FENCING_APP_PASSWORD} = get_var('QESAPDEPLOY_AZURE_SPN_APP_PASSWORD', get_required_var('_SECRET_AZURE_SPN_APP_PASSWORD'));
+    if (check_var('PUBLIC_CLOUD_PROVIDER', 'AZURE') || check_var('PUBLIC_CLOUD_PROVIDER', 'EC2')) {
```

Comment:
Same question as above.

Reply:
Removing the `if` here result in setting one more field in HASH `$variables{MAIN_ADDRESS_RANGE}` even if google will not use it later. This hash is later passed to `qesap_prepare_env`. I'll give it a try, I'm not sure what is going to happens :-)